### PR TITLE
initial systray support

### PR DESCRIPTION
**Pull request by zakius**
close and minimize to tray
double click tray icon to toggle window
tray context menu

### CODE REVIEW

**Comment by lorenzhs** [comment]
Hey zakius, thanks for implementing this, but it's quite hard to figure out what you did because you also converted all single spaces to double spaces. It's best to avoid such unnecessary changes.

File: electron-main.js [hunk 1]
```diff
@@ -232,27 +270,48 @@
...
+            {
+                label: "Show App", click: function () {
+                    mainWindow.toggle();
+                }
+            },
+            {
+                label: "Quit", click: mainWindow.quit
+            }
+        ]);
+        trayIcon.setToolTip("Electron.js App");
```
Comment (lorenzhs) [inline electron-main.js]: This should be "Glowing Bear"

**Comment by zakius** [comment]
Sorry for these extra changes, been doing this a few times due to problems with git workflow and it seems my IDE kicked in

**Comment by lorenzhs** [comment]
Closing since we removed electron in favour of tauri

**Pull request closed without merging**